packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -1470,39 +1472,47 @@ export class AccountHandler {
   async getAccount(request: AuthRequest) {
     this.log.begin('Account.get', request);
 
+    let playSubscriptions;
+    if (Container.has(PlaySubscriptions)) {
+      playSubscriptions = Container.get(PlaySubscriptions);
+    }
+
     const { uid, email } = request.auth.credentials;
 
-    let subscriptions: Awaited<MozillaSubscription[]> = [];
+    let webSubscriptions: Awaited<WebSubscription[]> = [];
+    let abbrevIapSubscriptions: Awaited<
+      { created: number; product_name: string }[]
+    > = [];
 
     if (this.config.subscriptions.enabled) {
       try {
-        if (this.stripeHelper) {
-          const customer = await this.stripeHelper.customer({
-            uid: uid as string,
-            email: email as string,
-          });
-          if (!customer) {
-            throw error.unknownCustomer(uid);
-          }
-          if (!customer.subscriptions) {
-            throw error.internalValidationError(
-              'Account.get',
-              { uid, customerId: customer.id },
-              'Expected customer to have subscriptions.'
-            );
-          }
```

Comment:
I don't throw any of these errors anymore, because an IAP subscriber with no web subscriptions wouldn't have a Stripe `customer`. I now check if `customer && customer.subscriptions` below.

Reply:
This depends on the consumers of this endpoint and what they expect. Previously we threw an error, where now we return an empty array for `subscriptions`.

How does FxA Settings use this endpoint? Is it checking for if the `subscriptions` object is on the customer?